### PR TITLE
Scalps & spelling

### PR DESCRIPTION
**Pull request by minervatbd**
Scalps are a cosmetic you get when you kill a player. Also there were some typos in nobite_text that Schrafft fixed.

### CODE REVIEW

File: ewcfg.py [hunk 1]
```diff
@@ -10109,35 +10109,36 @@ def wef_bass(ctn = None):
...
+    	"♪ That Captain Albert Alexander! ♪",
+    	"You get the urge to jump in and try to grab a fish, before remembering that you can't swim...",
+    	"You hum some sea shanties...",
+    	"You start to slip into an existential crisis...",
+    	"You jitter as other seamen catch fish before you. Fuck fishing...",
+    	"You feel the oncoming downward spiral...",
+    	"You shake your head as a young seaman baits a perfectly good slice of pizza on his hook... What a cretin...",
+    	"You wonder if the Space Navy has been formed yet...",
+    	"Man... Why were you excited for this shit?",
+    	"Still better than Minesweeper...",
```
Comment (TrackpadSnake) [inline ewcfg.py:11413]: please remove this one, it makes me sad

Comment (minervatbd) [reply ewcfg.py:11413]: never

File: ewwep.py [hunk 2]
```diff
@@ -1178,3 +1192,32 @@ def explode(damage = 0, district_data = None):
 
 		await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 
+def scalp_text(id_weapon):
```
Comment (magicksRampage) [inline ewwep.py]: makes these attributes in the EwWeapon class instead

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewwep.py [hunk 3]
```diff
@@ -724,6 +743,25 @@ def canAttack(cmd):
...
+						item_type = ewcfg.it_cosmetic,
+						id_user = cmd.message.author.id,
+						id_server = cmd.message.server.id,
+						item_props = {
+							'id_cosmetic': 'scalp',
+							'cosmetic_name': "{}'s scalp".format(shootee_name),
+							'cosmetic_desc': "A scalp.{}".format(scalp_text),
+							'adorned': 'false'
+						}
+					)
```
Comment (magicksRampage) [inline ewwep.py:751]: why is the scalp being created twice?

File: ewwep.py [hunk 4]
```diff
@@ -724,6 +743,25 @@ def canAttack(cmd):
 
 					#add bounty
 					user_data.add_bounty(n = (shootee_data.bounty / 2) + (slimes_dropped / 4))
+          
+          # Scalp text
+          if weapon != None:
+            scalp_text = weapon.str_scalp
+          else:
+            scalp_text = ""
```
Comment (magicksRampage) [inline ewwep.py]: this part is not properly formatted.

**Review by magicksRampage — CHANGES_REQUESTED** [review]

**Pull request merged**